lib/tasks/demo.rake

```diff
@@ -380,19 +415,19 @@ namespace :demo do
     end
 
     Patient.import! patients
-    new_patients = Patient.where('created_at >= ?', today)
-    new_patients.update_all('responder_id = id')
+    new_patientients = Patient.where('created_at >= ?', today)
```

Comment:
couple of typos here?

Reply:
🤦 Yup. Not sure why this did not throw an error when I ran the rake task. Good catch

Reply:
Ah, it didn't throw an error because new_patientients is actually defined -- Was a copy+paste error